<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewRecipeWindow</class>
 <widget class="QMainWindow" name="NewRecipeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="batch_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Batch n°1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Name :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Date :</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="date_box">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Volume</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="volume_box">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>10000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>130</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Liters</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>170</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>°</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="alcohol_box">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Predicted Alcohol</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="sugar_box">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>210</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>210</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>gr/L</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Residual Sugar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_box">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>411</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Honey Quantity</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="honey_box">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>290</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>1000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>290</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>gr</string>
    </property>
   </widget>
   <widget class="QPushButton" name="honey_button">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>250</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate Honey</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Yeast</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>140</y>
      <width>250</width>
      <height>151</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>250</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>205</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>200</number>
    </attribute>
    <row>
     <property name="text">
      <string>Dry</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Off-Dry</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Sweet</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Dessert</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Residual Sugar (gr/L)</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>0 - 10</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>10 - 20</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>20 - 40</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string>40 - 60</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Fermaid K</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="fermaid_box">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>370</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="save_box">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>530</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Recipe</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="note_box">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>330</y>
      <width>251</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
